<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>33</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>306</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Turns</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="turns_bpm">
      <item>
       <property name="text">
        <string>bpm01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bpm17</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_inj">
      <property name="text">
       <string>INJ ALL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_free">
      <property name="text">
       <string>FREE ALL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
